mantidproject/mantid
# Examples for creating Custom ColorCycles and ColorMaps

**Description of work**

Writing some example scripts and instructions for plotting with a Custom ColorCycle and creating Custom Colormaps.

Fixes #24485 
refs #27503

**To test:**

Build docs.html, check the changes look correct and the examples work!

*This does not require release notes* because **it's Documentation**
---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
